src/classes/ADDR_Contact_TDTM.cls

```diff
@@ -60,16 +61,15 @@ public class ADDR_Contact_TDTM extends TDTM_Runnable {
     
     /*******************************************************************************************************
     * @description Trigger Handler on Contact that handles Address Management.
-    *
     *   Rules:
     *       inserting new contact
     *           if parent Account is Household --> create new Address as child of Account. If not override, make it default and
-                                                   propagate to children without override
+                                                    propagate to children without override
```

Comment:
@MatthewBlanski , thanks for catching that! I've updated and added *. Btw, the way the rules are written is not something I'm 100% found off. However, I didn't want to make the update since I'm not really refactoring ADDR_Contact_TDTM and I know we have it in our list of candidates to refactor in the future.